babel.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  presets: [['@shopify/babel-preset', {typescript: true}]],
+  overrides: [
+    {
+      test: './polaris-react',
+      presets: [['@shopify/babel-preset', {typescript: true, react: true}]],
+    },
+  ],
```

Comment:
This approach looks good to me!

One "unfortunate" caveat is we have to run `yarn build` from the monorepo root for local development. From your [linked docs](https://babeljs.io/docs/en/config-files#root-babelconfigjson-file):

>The first issue you'll likely run into is that by default, Babel expects to load [babel.config.json](https://babeljs.io/docs/en/config-files#project-wide-configuration) files from the directory set as its ["root"](https://babeljs.io/docs/en/options#root), which means that if you create a [babel.config.json](https://babeljs.io/docs/en/config-files#project-wide-configuration), but run Babel inside an individual package the ["root"](https://babeljs.io/docs/en/options#root) Babel is using in that context is not your monorepo root, and it won't be able to find the [babel.config.json](https://babeljs.io/docs/en/config-files#project-wide-configuration) file.

I often change into package directories for my development workflow and without a local `<package>/.babelrc.json` with `rootMode: 'upward'` the build won't work. The docs provide a few examples of various ways you can specify `rootMode: 'upward'`, not sure how that works with loom, but with all that said I wouldn't considered this edge case a show stopper!

Reply:
> One "unfortunate" caveat is we have to run yarn build from the monorepo root for local development.

Running the loom build from the monorepo root - which would run the build for all loom projects - of which there is just the one -  is very easy and is a minimal code change. That said https://github.com/Shopify/polaris/blob/main/polaris-react/loom.config.ts does not override any babel config and by default [loom-plugin-build-library disables looking at babel.config.js for babel config](https://github.com/Shopify/loom/blob/main/packages/loom-plugin-build-library/src/plugin-build-library.ts#L47-L48), so I don't think the placement of this file has any effect for the polaris-react build.


Because of that I don't understand why the overrides section is needed here - Loom shouldn't be reading this file given its current configuration state.